### ISSUE TITLE
Flashing help

### ISSUE THREAD

**Issue by casperround** [gearsincorg/Hoverboards-for-assistive-devices#3]
When flashing do you have to flash both boards? I've flashed the main master successful, powered on and the slave board operates like stock. So wondering if I need to flash that side too??

**Comment by gearsincorg** [comment]
Yeah.  I posted answer on Youtube...    You need to flash both boards the same and then yuo can send commands to both of them, or just the master.  See the ESP32 code for examples of sending to both boards, or se the PIC BluJoy code for examples of commands just to the Master.

**Comment by casperround** [comment]
> Yeah.  I posted answer on Youtube...    You need to flash both boards the same and then yuo can send commands to both of them, or just the master.  See the ESP32 code for examples of sending to both boards, or se the PIC BluJoy code for examples of commands just to the Master.

Thank you, I just saw I will give this a go

**Comment by rohitsahu7** [comment]
Hello, thanks for this awesome project , I want to make it too, I have complied the firmware in Keil , I also have the same question. 
1. Shall I flash the same code in both the Master and slave boards?
2. How does the code identify which one is master?
3. Can u provide the link of ESP32 code , Is it .ino file?
4. I want to use Arduino Mega controller for this project can you help me please
can you mail me at rohit.kr.sahu@gmail.com

Thanks and regard
Rohit

**Comment by gearsincorg** [comment]
1) Yes, the code goes into both boards.

2) They don't need to know which is master and which is slave...  If you are sending motion commands in through the plug-in serial ports (into each controller from the ESP32), they each just do what you tell them.    

3) Yes, the .ino file and the other in that folder are the ESP32 code (arduino IDE)

4) If you use the Arduino Mega, you can probably use very simmilar code to the ESP32.  You will need to setup two serial ports (1 for master and 1 for slave)    Look at the protocol doc in the HUGS doc folder, and compare it to the ESP and Keil Comms files.

I can't really provide detailed support, everyone's application and hardware may be different.
